### PR TITLE
Add initial draft of a Remote Access solutions guide for TrueNAS 24.10

### PR DESCRIPTION
**Pull request by kmoore134**
Thanks for contributing to TrueNAS documentation! By opening a Pull Request, you're acknowledging that your changes will be distributed under the [Creative Commons 4.0](https://creativecommons.org/licenses/by-nc-sa/4.0/) license.

### CODE REVIEW

File: content/Solutions/RemoteAccess.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+## Overview
+
+TrueNAS is fundamentally a storage appliance, and as such it is recommended to always be run not directly exposed to the internet.
+
+However, there are many instances in which a user will need to access their storage data or services remotely. In most corporate
+environments this is controlled by having access to a corporate VPN service where proper auditing and security best practices
+can be enforced. For the small business or home lab where a corporate VPN isn't available, there are many easier to deploy and
+manage services that can be used to give yourself or your clients access to your storage services and applications. This guide
+will introduce you to several of these and provide a basic walk-through of how to enable remote access to your TrueNAS.
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:20]: Suggested rewording to make this active voice/present tense, remove contractions, and to make the text more concise.
```suggestion
However, there are many instances where a user needs to access their storage data or services remotely. 
In most corporate environments, a corporate VPN service controls internal and external access, performs proper auditing, and applies and enforces security best practices. 
For the small business or home lab, where a corporate VPN is not an option, you can choose from many easy-to-deploy and manage services that give you or your clients access to your storage services and applications. 
This guide introduces several access options and provides a basic walk-through enabling remote access to your TrueNAS.
```

File: content/Solutions/RemoteAccess.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+In TrueNAS 24.10 and later, there are several VPN technologies to choose from, each with their own strengths and weaknesses that
+may make them suitable for your particular environment. We will focus on three of them, all of which are available via the 
+TrueNAS App catalog for rapid deployment.
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:24]: Same reason for these recommended changes:
```suggestion
In TrueNAS 24.10 and later, there are several VPN technologies to choose from, each with strengths and weaknesses that make them suitable for particular environments. 
This guide focuses on three options available through the TrueNAS App catalog for rapid deployment. 
```

File: content/Solutions/RemoteAccess.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+- ZeroTier
+
+- Wireguard
+
+## Tailscale
+
+Website: [Tailscale](https://www.tailscale.com/)
+Pricing: Free / Paid Tier
+Difficulty: Easy to Moderate
+Usability: Tailscale focuses on simplicity and ease of setup. It uses a centralized control plane and integrates with identity providers like Google or Microsoft for authentication. It’s ideal for users with limited networking knowledge.
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:37]:
```suggestion
Usability: Tailscale focuses on simplicity and ease of setup. It uses a centralized control plane and integrates with identity providers like Google or Microsoft for authentication. It is ideal for users with limited networking knowledge.
```

File: content/Solutions/RemoteAccess.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+### Installing Tailscale on TrueNAS:
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:39]:
```suggestion
### Installing Tailscale in TrueNAS:
```

File: content/Solutions/RemoteAccess.md [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+
+Before you install Tailscale on TrueNAS, you will need to create an [Auth Key](https://login.tailscale.com/admin/settings/keys) via your Tailscale account. [More Information](https://tailscale.com/kb/1085/auth-keys)
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:41]:
```suggestion
Before installing Tailscale in TrueNAS, create an [Auth Key](https://login.tailscale.com/admin/settings/keys) through your Tailscale account. [More Information](https://tailscale.com/kb/1085/auth-keys)
```

File: content/Solutions/RemoteAccess.md [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+
+Once your Auth Key has been created, you can navigate to the TrueNAS Apps Catalog and install Tailscale:
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:43]:
```suggestion
After creating your Auth Key you can navigate to the TrueNAS Apps Catalog and install Tailscale:
```

File: content/Solutions/RemoteAccess.md [hunk 7]
```diff
@@ -0,0 +1,89 @@
...
+
+Apps -> Discover Apps -> (Search for Tailscale) -> Install
+
+On the Installation page, scroll down to the `AuthKey` field, paste in the key you created previously. Leave all other options at their defaults, then scroll down and click `Install`.
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:47]: Fixed emphasis, changing it from code to bold, and suggested rewording to make the text concise.
```suggestion
On the Installation page, scroll down to the **AuthKey** field, and paste the previously created key. Accept all other default settings, and then scroll down and click **Install**.
```

File: content/Solutions/RemoteAccess.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+
+Once the installation completes and you see the App status as "Running" you can navigate back to your [Tailscale Machines](https://login.tailscale.com/admin/machines) list and you will see your TrueNAS system listed.
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:49]: We don't say "once", we use after or when based on how something is worded.  Suggestions to make active voice/present tense and other text changes
```suggestion
After the installation completes and you see the app status as **Running** you can navigate back to your [Tailscale Machines](https://login.tailscale.com/admin/machines) list to see your TrueNAS system listed.
```

File: content/Solutions/RemoteAccess.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+
+Using the `Addresses` listed for TrueNAS, you can now reach your TrueNAS host, including Shares, Apps and other services from your other [Tailscale clients](https://tailscale.com/download) anywhere in the world.
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:51]:
```suggestion
Using the **Addresses**  listed for TrueNAS, you can now reach your TrueNAS host, including shares, apps, and other services from your other [Tailscale clients](https://tailscale.com/download) anywhere in the world. 
```

File: content/Solutions/RemoteAccess.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+Using the `Addresses` listed for TrueNAS, you can now reach your TrueNAS host, including Shares, Apps and other services from your other [Tailscale clients](https://tailscale.com/download) anywhere in the world. 
+
+
+
+## ZeroTier
+
+Website: [ZeroTier](https://www.zerotier.com/)
+Pricing: Free / Paid Tier
+Difficulty: Moderate to Hard
+Usability: While still being user-friendly, ZeroTier offers more advanced configuration options, such as managing multiple networks, IP ranges, and multicast settings. This makes it better suited for users with technical expertise.
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:60]:
```suggestion
Usability: User-friendly ZeroTier offers more advanced configuration options, such as managing multiple networks, IP ranges, and multicast settings. This makes it better suited for users with technical expertise.
```

File: content/Solutions/RemoteAccess.md [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+
+### Installing ZeroTier on TrueNAS:
+
+Before you install ZeroTier on TrueNAS, you will need to create an [Network](https://my.zerotier.com/network) via your ZeroTier account. [More Information](https://docs.zerotier.com/start)
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:64]:
```suggestion
Before installing ZeroTier on TrueNAS, create a [Network](https://my.zerotier.com/network) through your ZeroTier account. [More Information](https://docs.zerotier.com/start)
```

File: content/Solutions/RemoteAccess.md [hunk 12]
```diff
@@ -0,0 +1,89 @@
...
+
+Once your Network has been created, copy the `Network ID` and then you can navigate to the TrueNAS Apps Catalog to install ZeroTier:
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:66]:
```suggestion
After creating the Network, copy the **Network ID** and then navigate to the TrueNAS Apps Catalog to install ZeroTier:
```

File: content/Solutions/RemoteAccess.md [hunk 13]
```diff
@@ -0,0 +1,89 @@
...
+
+Apps -> Discover Apps -> (Search for ZeroTier) -> Install
+
+On the Installation page, scroll down to the `ZeroTier Configuration -> Networks` field. Click to add a network and paste in the network key you copied previously. Leave all other options at their defaults, then scroll down and click `Install`.
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:70]:
```suggestion
On the installation screen, scroll down to the **ZeroTier Configuration -> Networks** field.
Click to add a network and paste in the network key you copied previously. Accept all other default settings, and then scroll down to click **Install**.
```

File: content/Solutions/RemoteAccess.md [hunk 14]
```diff
@@ -0,0 +1,89 @@
...
+
+Once the installation completes and you see the App status as "Running" you can navigate back to your [ZeroTier Networks](https://my.zerotier.com/networks) list, click your network and you will see your TrueNAS system waiting for approval.
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:72]:
```suggestion
After the installation completes and the App status changes to "Running", you can navigate back to your [ZeroTier Networks](https://my.zerotier.com/networks) list, and click on your network to see your TrueNAS system waiting for approval.
```

File: content/Solutions/RemoteAccess.md [hunk 15]
```diff
@@ -0,0 +1,89 @@
...
+
+Click to authorize TrueNAS and you will have successfully finished joined your TrueNAS system to a ZeroTier network.
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:74]:
```suggestion
Click to authorize TrueNAS and you have successfully finished and joined your TrueNAS system to a ZeroTier network.
```

File: content/Solutions/RemoteAccess.md [hunk 16]
```diff
@@ -0,0 +1,89 @@
...
+
+Using the `Addresses` listed for TrueNAS, you can now reach your TrueNAS host, including Shares, Apps and other services from your other [ZeroTier clients](https://www.zerotier.com/download/) anywhere in the world.
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:76]:
```suggestion
Using the **Addresses** listed for TrueNAS, you can reach your TrueNAS host, including shares, apps, and other services from your other [ZeroTier clients](https://www.zerotier.com/download/) anywhere in the world. 
```

File: content/Solutions/RemoteAccess.md [hunk 17]
```diff
@@ -0,0 +1,89 @@
...
+## Wireguard
+
+Website: [WireGuard](https://www.wireguard.com)
+Pricing: Free
+Difficulty: Hard
+Usability: Wireguard requires manual setup and configuration. You need to manage keys, IPs, and routing yourself.
+
+### Installing Wireguard on TrueNAS:
+
+Wireguard can be setup and configured by installing the `WG Easy` Application in the TrueNAS app catalog. [Click Here](https://www.truenas.com/docs/truenasapps/stableapps/installwgeasyapp/) for more detailed information on how to setup and configure.
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:89]:
```suggestion
Wireguard can be setup and configured by installing the **WG Easy** application in the TrueNAS app catalog. [Click here](https://www.truenas.com/docs/truenasapps/stableapps/installwgeasyapp/) for more detailed information on  setting up and configuring the app.
```

File: content/Solutions/RemoteAccess.md [hunk 18]
```diff
@@ -0,0 +1,89 @@
...
+Difficulty: Easy to Moderate
+Usability: Tailscale focuses on simplicity and ease of setup. It uses a centralized control plane and integrates with identity providers like Google or Microsoft for authentication. It’s ideal for users with limited networking knowledge.
+
+### Installing Tailscale on TrueNAS:
+
+Before you install Tailscale on TrueNAS, you will need to create an [Auth Key](https://login.tailscale.com/admin/settings/keys) via your Tailscale account. [More Information](https://tailscale.com/kb/1085/auth-keys)
+
+Once your Auth Key has been created, you can navigate to the TrueNAS Apps Catalog and install Tailscale:
+
+Apps -> Discover Apps -> (Search for Tailscale) -> Install
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:45]:
```suggestion
**Apps -> Discover Apps ->** (Search for Tailscale) **-> Install**
```

File: content/Solutions/RemoteAccess.md [hunk 19]
```diff
@@ -0,0 +1,89 @@
...
+Difficulty: Moderate to Hard
+Usability: While still being user-friendly, ZeroTier offers more advanced configuration options, such as managing multiple networks, IP ranges, and multicast settings. This makes it better suited for users with technical expertise.
+
+### Installing ZeroTier on TrueNAS:
+
+Before you install ZeroTier on TrueNAS, you will need to create an [Network](https://my.zerotier.com/network) via your ZeroTier account. [More Information](https://docs.zerotier.com/start)
+
+Once your Network has been created, copy the `Network ID` and then you can navigate to the TrueNAS Apps Catalog to install ZeroTier:
+
+Apps -> Discover Apps -> (Search for ZeroTier) -> Install
```
Comment (micjohnson777) [inline content/Solutions/RemoteAccess.md:68]:
```suggestion
**Apps -> Discover Apps ->** (Search for ZeroTier) **-> Install**
```

**Review by micjohnson777 — CHANGES_REQUESTED** [review]
Sorry, I bled red ink all over this. You can fire me later. ;-)

**Comment by kmoore134** [comment]
> Sorry, I bled red ink all over this. You can fire me later. ;-)

Ha, no worries. I was sure it needed fixing. Thank you and ready for your review again.

**Pull request merged**